src/lang/bg-BG.json

```diff
@@ -866,4 +866,4 @@
   "region_15":"Перу",
   "region_16":"Индия",
   "region_19":"Япония"
-}
\ No newline at end of file
+}
```

Comment:
Can you also revert these newline adds?  might conflict with the localization PR.